_data/tags.yml

```diff
@@ -3,6 +3,7 @@ tags:
 - action-recognition
 - active-learning
 - adversarial
+- attention
```

Comment:
Not sure `attention` on its own fits as a keyword that is self-explaining, especially having already the `attention-maps` keyword. May be it could be used to propose a more complete keyword (I cannot imagine which, though)?

Reply:
I just searched the repo and found that the `attention` tag was used before by Fred. I think it has its place since it is a buzzword and an approach that is becoming more and more common.
Personally, I find `attention-maps` too specific. If I want to search for articles about attention applied to CNNs, I would search "cnn attention", or something. I would never have though of searching  for "attention-maps". Anyhow, I think we should keep both for now.

Reply:
OK.